.github/workflows/podvm.yaml

```diff
@@ -2,42 +2,27 @@ name: Create Pod VM Image
 on:
   workflow_run:
     workflows: ["Create Pod VM Builder Image"]
-    types: 
+    types:
       - completed
 
 jobs:
   build:
     name: Create pod vm image
     if: ${{ github.event.workflow_run.conclusion == 'success' }}
-    runs-on: ${{ matrix.runner }}
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        os:
-          # Please keep this list in alphabetical order.
-          - centos
-          - ubuntu
-        provider:
-          # Please keep this list in alphabetical order.
-          - aws
-          - azure
-          - ibmcloud
-          - libvirt
-          - vsphere
-        arch:
-          - amd64
-        include:
+        os: [centos, ubuntu]
+        arch: [amd64, s390x]
+        provider: [generic, vsphere]
+        exclude:
           - os: centos
-            dockerfile: Dockerfile.podvm.centos
```

Comment:
@jtumber-ibm don't you need to pass the dockerfile in https://github.com/confidential-containers/cloud-api-adaptor/pull/660/commits/e30241fd02096f07cc90c2f021c69cce84064597#diff-0721be05d0601b7a1d79abe8aca4a7682f18f6ef15b7505b5951ce10bc1ef072R53 ?

Reply:
Good catch! Yes I do.

```
act -n -j build -W .github/workflows/podvm.yaml | grep Matrix
*DRYRUN* [Create Pod VM Image/Create pod vm image-4] 🧪  Matrix: map[arch:s390x dockerfile:Dockerfile.podvm os:ubuntu provider:generic]
*DRYRUN* [Create Pod VM Image/Create pod vm image-3] 🧪  Matrix: map[arch:amd64 dockerfile:Dockerfile.podvm os:ubuntu provider:vsphere]
*DRYRUN* [Create Pod VM Image/Create pod vm image-1] 🧪  Matrix: map[arch:amd64 dockerfile:Dockerfile.podvm.centos os:centos provider:generic]
*DRYRUN* [Create Pod VM Image/Create pod vm image-2] 🧪  Matrix: map[arch:amd64 dockerfile:Dockerfile.podvm os:ubuntu provider:generic]
```